elated-egrets/bangazon-api
# Added a no delete feature to the User

## DESCRIPTION

## How to pull down
Commands

```
git fetch --all
git pull JRuserdel
git checkout JRuserdel
python manage.py runserver
```

## Steps to test
pull the code down and run the server
make sure that there are no errors and you see the no delete feature in vs code 
## What you changed
added the no delete feature to the user model
## What was added to read me
n/a
## Final Checklist
Before merging.
1. Has each file changed been reviewed?
1. Has the readme been updated to reflect changes made?
1. Has a teammate approved the PR?

Then Ship It!
